Makefile

```diff
@@ -115,15 +115,15 @@ endif
 
 build-linux:
 	mkdir -p $(BUILDDIR)
-	docker build --no-cache --tag classic-terra/core ./
-	docker create --name temp classic-terra/core:latest
+	docker build --platform linux/amd64 --tag classic-terra/core ./
```

Comment:
--no-cache is important here if we need to update docker to newest source, I use it somewhat often when I modify source code

Reply:
Got it. I will revert it.

Reply:
I actually considered the --no-cache option as well and I agree with Vinh short-term. However long term I think what makes more sense is to implement some sort of "hot reloading" feature using something like https://github.com/cosmtrek/air and method #2 as explained here @ https://techinscribed.com/5-ways-to-live-reloading-go-applications/. This will actually allow us to rebuild the code without having to rebuild the image all the time, as --no-cache is like a hammer that will break all your intermediate layers including apk/apt packages and/or genesis downloading, which does eat up a significant amount of resources during the build process, especially if your building more then just the terrad container image (E.g. the full node image)

Reply:
@ZaradarBH Thanks for the input. Let's make a backlog issue for it.

Reply:
I want to try implementing that "hot reloading"